.github/workflows/dev.yml

```
@@ -2,7 +2,7 @@ name: Release
 
 on:
   branches:
```

actionlint (1.7.7)

unknown Webhook event "branches". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)